src/dune_rules/file_binding.mli

```diff
@@ -41,5 +41,7 @@ module Unexpanded : sig
 
   module L : sig
     val decode : t list Dune_lang.Decoder.t
+
+    val find_var : t list -> Loc.t option
```

Comment:
Could you add a docstring to this function? The name does not make its behavior obvious.